src/config.rs

```diff
@@ -20,141 +20,129 @@ const PLAIN: u32 = 0;
 const SALSA_20: u32 = 2;
 const CHA_CHA_20: u32 = 3;
 
+/// Choices for outer encryption
 #[derive(Debug, Clone)]
 #[cfg_attr(feature = "serialization", derive(serde::Serialize))]
-pub enum OuterCipherSuite {
+pub enum OuterCipherConfig {
     AES256,
     Twofish,
     ChaCha20,
 }
 
+/// Errors for outer encryption configuration
 #[derive(Debug, Error)]
-pub enum OuterCipherSuiteError {
+pub enum OuterCipherConfigError {
```

Comment:
@sseemayer could be only `OuterCipherError`?

Reply:
Since the error is not with the cipher itself, but with its configuration, I would prefer to keep the name as-is.